File: java_euler/p001.java
/**
 * Project Euler - Problem 1
 * Find the sum of all the multiples of 3 or 5 below 1000
 * 
 * p001.java
 * javac p001.java
 * java p001
 */

public class p001 {
    public static void main(String[] args) {
	System.out.println(new p001().run());
    }

    public String run() {
	int sum = 0;
	for (int i = 0; i < 1000; i++){
	    if (i%3 == 0 || i%5 == 0)
		sum += i;
	}
	return Integer.toString(sum);
    }
}

File: cpp_euler/0-20/p001.cpp
/* Project Euler - Problem 1
  Find the sum of all the multiples of 3 or 5 below 1000
  
  p001.cpp
  g++ p001.cpp -o p1.exe
*/
#include <iostream>

int main() {
  int sum = 0;
  for (int i = 1; i < 1000; i++)
    if ((i % 3 == 0) || (i % 5 == 0))
      sum+=i;

  std::cout << sum << std::endl;
  return 0;
}
